packages/plugins/model-train/cycle-gan-tensorflow-model-train/image_loader/image_loader.py

```diff
@@ -0,0 +1,39 @@
+import cv2
```

Comment:
I don't like the name "cycle-gan-tensorflow-model-train", we need to figure out a more general one.